articles/accounts_certified_users_and_qualified_researchers.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: "Accounts, Certified Users and Qualified Researchers"
+title: "Synapse User Credentials including Certified Users and Qualified Researchers"
 layout: article
```

Comment:
I am not sure User credentials help as a concept here; it could easily be confused the sharing settings which is often callled user credentials in software systems.  Is there another more neutral term we could use?